### DISCUSSION TITLE
How it works ?

### DISCUSSION THREAD

**Discussion by Mostafa-Mohammadi** [timlrx/tailwind-nextjs-starter-blog#562] [General]
Does this project use a CMS to fetch blogs from there? Something like  Strapi or Contentful?

Please explain how it works,  I want to use the ISR feature from Next js for my blog and I don't have any idea how to put it inside this starter.

**Comment by SrikanthKumarC** [comment]
This uses .mdx and .md files which are inside `data/blog` folder. Learn more about MDX at https://mdxjs.com. 

I am not sure about the ISR thing, so I can't talk about it.
